.github/workflows/lint.yml

```
@@ -20,6 +20,6 @@ jobs:
           node-version: ${{ env.NODE_VERSION }}
           cache: "npm"
       - name: Install dependencies
-        run: npm install
+        run: npm ci
```

```suggestion
        run: npm install
```

I _think_ we wanna keep this as install - `ci` is important for the build process, but not really for eslint, where we're happy for the "other stuff" to happen when we install

If we want to verify that `npm ci && npm run build` work as expected I think it's valuable to add that as an additional workflow. For example, at work I setup:

```
- name: Install dependencies - early feedback on production install
  run: npm ci --production
- name: Build
  run: npm run build
```